lib/mvgApi.js

```diff
@@ -15,36 +15,36 @@ const iconv = require("iconv");
  */
 function getDepartures(station, options) {
 
-    let transitParams = buildTransitParams(options);;
+    let transitParams = buildTransitParams(options);
 
-    const requestUrl = 'http://www.mvg-live.de/ims/dfiStaticAuswahl.svc?haltestelle='.concat(station).concat(transitParams);
+    const requestUrl = 'https://cors-anywhere.herokuapp.com/https://www.mvg-live.de/ims/dfiStaticAuswahl.svc?haltestelle='.concat(station).concat(transitParams);
```

Comment:
Change this to:
`const requestUrl = apiUrl.concat('?haltestelle').concat(station).concat(transitParams);`